app/classifier/tasks/text/index.cjsx

```diff
@@ -58,22 +58,12 @@ module.exports = React.createClass
     annotation: null
     onChange: NOOP
 
-  render: ->
-    <GenericTask question={@props.task.instruction} help={@props.task.help} required={@props.task.required}>
-      <label className="answer">
-        <textarea autoFocus={@props.autoFocus} className="standard-input full" rows="5" ref="textInput" value={@props.annotation.value} onChange={@handleChange} />
-      </label>
-      {if @props.task.text_tags 
-          <div className="transcription-metadata-tags">
-            {for tag, i in @props.task.text_tags
-              <input type="button" className="standard-button text-tag" key={i} value={tag} onClick={@setTagSelection} />}
-          </div>}
-    </GenericTask>
+  getInitialState: ->
+    textareaHeight: undefined
+    initOffsetHeight: undefined
```

Comment:
What's the relevance of the textarea's initial height? Its height should be set automatically on mount.